features/org.mozilla.xulrunner.feature/pom.xml

```diff
@@ -26,6 +26,11 @@
 						</goals>
 					</execution>
 				</executions>
+				<configuration>
+					<excludes>
+						<plugin id="org.mozilla.xulrunner.cocoa.macosx"/>
```

Comment:
Yes, but it doesn't explain why the Linux stuff is missing. :{

I added the exclude to the `source-feature` goal because the source feature build was failing to find sources for o.m.x.cocoa.macosx. It should not prevent the BUILDING of the plugins, only the autogen of the source plugins in the source feature.